release-tools/generate-release-candidate.py

```diff
@@ -0,0 +1,201 @@
+#! /usr/bin/python
+"""
+About:
+               Copy the Staging Artifiacts from Snapshots to Release Candidate folder 
+
+Prerequisites:
+               Need python 3.7 or higher
+               AWS credentials for S3
+               sudo yum install python-pip
+               sudo pip install boto3
+               sudo pip install pyyaml
+
+Usage: 
+              ./copy_snapshots_to_release.py $BUILD_NUMBER
```

Comment:
Shouldn't it be ./generate-release-candidate.py

Reply:
Sorry, my bad. Will change it.

Reply:
Change made.